dbdialog/src/org/pentaho/ui/database/DatabaseConnectionDialog.java

```diff
@@ -34,7 +34,7 @@
 
   public static final String DIALOG_DEFINITION_FILE = "org/pentaho/ui/database/databasedialog.xul";
 
-  private Map<String, String> extendedClasses = new HashMap<String, String>();
+  protected Map<String, String> extendedClasses = new HashMap<>();
```

Comment:
The diamond operator is a Java 7 feature, @dkincade said we can use Java 7 features in master now